src/js/actions/index.js

```diff
@@ -679,3 +681,11 @@ export const fetchPreferenceTypes = (config) => async (dispatch) => {
     payload: preferenceTypes?.data?.data,
   });
 };
+
+export const fetchRatingTypeCodes = () => async (dispatch) => {
+  const ratingTypeCodes = await apiClient.get(RATING_TYPE_OPTIONS);
```

Comment:
I feel like we should use our API services
(I mean those services :point_down:)
![image](https://github.com/openboxes/openboxes/assets/83239466/e18422d6-ca35-4071-a0b7-796e69230f17)

Reply:
done